airflow_account/models/account_invoice.py

```diff
@@ -22,17 +22,28 @@ class AccountInvoice(models.Model):
     paid_date = fields.Date(string='Paid Date')
 
     sale_id = fields.Many2one('sale.order',compute="_compute_sale_id", string="Original Sale Order",store=True)
-    proj_manager = fields.Many2one('res.partner', related='sale_id.proj_manager', string="Project Manager", default=False, store=True)
-    proj_name = fields.Char(related='sale_id.proj_name', string="Project Name", default=False, store=True)
-    ship_date = fields.Date(related='sale_id.ship_date', string="Ship Date", default=False, store=True)
-    ship_method = fields.Many2one('delivery.carrier', related='sale_id.ship_method', string="Shipping Method", default=False, store=True)
+
+    # The field project_manager is depricated. Airflow wishes to have it hold Char type instead of many2one.
+    # Keeping field for historical reasons and data preservation.
+    proj_manager = fields.Many2one('res.partner', related='sale_id.proj_manager', string="Project Manager(Depricated)", store=True)
+
+    proj_manager_char = fields.Char(related='sale_id.proj_manager_char', string="Project Manager", store=True)
```

Comment:
Make field name to end with _name instead of _char
```suggestion
    proj_manager_name = fields.Char(related='sale_id.proj_manager_name', string="Project Manager", store=True)
```